### ISSUE TITLE
feat(YouTube): Remove feed playlists

### ISSUE THREAD

**Issue by Bluigi** [ReVanced/revanced-patches#69]
### Feature description

On the home section of YouTube I get playlists of music i didn't create. I was hoping it would be possible to remove those.
https://imgur.com/a/rQG5nuk

### Motivation

It's annoying and it's always music I don't care about.

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by daveout** [comment]
Are those "mix playlists"?

**Comment by oSumAtrIX** [comment]
Yes, such a patch exists already, but it does not reliably hide every playlist. A bug report should already exist.
